xml/admin_nfsganesha.xml

```diff
@@ -484,10 +483,10 @@ EXPORT {
     <listitem>
      <para>
       <emphasis>Only full, sequential write I/O is supported.</emphasis>
-      Therefore, write operations are forced to be uploads. Many typical I/O
-      operations, such as editing files in place, will necessarily fail as they
-      perform non-sequential stores. There are file utilities that apparently
-      write sequentially (for example, some versions of GNU
+      All write operations should be treated as an upload. Many typical I/O
+      operations, such as editing files in place, will fail because they
+      perform non-sequential stores. There are file utilities that preform
```

Comment:
```suggestion
      perform non-sequential stores. There are file utilities that perform

Reply:
Presumably you meant `perform` here, @mgfritch ?

Reply:
uhm yep :+1: